sdk/core/Azure.Core/tests/LowLevelClientTests.cs

```diff
@@ -211,6 +212,25 @@ await client.GetPetAsync("snoopy", new RequestContext()
             });
         }
 
+        [Test]
+        public async Task Change404Category()
+        {
+            var mockResponse = new MockResponse(404);
+
+            // Send the response through the pipeline so IsError is set.
+            var mockTransport = new MockTransport(mockResponse);
+            PetStoreClient client = CreateClient(mockTransport);
+
+            #region Snippet:SetRequestContext
+            var context = new RequestContext();
+            context.AddClassifier(404, isError: false);
+
+            Response response = await client.GetPetAsync("pet1", context);
```

Comment:
Ah, I see where the issue is now.  We should be able to regenerate or manually update PetStoreClient to bring it up to date with the rest of the generated clients.

Reply:
I've made changes in `PetStoreClient` to be more aligned with a generated client. The test case failure is fixed.